rdr_client/work_queue.py

```diff
@@ -1,18 +1,50 @@
-""" Uses the service account 'awardee-pitt' to get participant summaries for work queue api
-    with an awardee of PITT"""
+""" Uses a service account to get and sync participant summaries for work queue api
+    You can only return results for the same awardee that the service account is connected to.
+
+    Run this using ./run_client.sh --project all-of-us-rdr-prod --account [your account]
+    --service_account [service account]@all-of-us-rdr-prod.iam.gserviceaccount.com work_queue.py
```

Comment:
I would remove @all-of-us-rdr-prod.iam.gserviceaccount.com; that's a part of the service account, and might vary depending on how we do things. (You could give an example here afterwards.)

Reply:
done